src/main/java/com/ericsson/eiffelcommons/subscriptionobject/MailSubscriptionObject.java

```diff
@@ -14,4 +16,14 @@ public MailSubscriptionObject(String subscriptionName) throws IOException {
 
         subscriptionJson.put("notificationType", "MAIL");
     }
+
+    /**
+     * Sets the field emailSubject to given value
+     * @param emailSubject
```

Comment:
param name mismatches the object in the parameter.

Reply:
Also can we add proper javadoc for the parameter and return type.

Reply:
It returns itself, I added the class type. None of the other functions have description of parameters. Probably because they are so simple. A description would go something along the line of '@param emailSubject the subject of the email', which is quite redundant.